OWNERS

```diff
@@ -3,12 +3,14 @@ approvers:
 - wongma7
 - jyotimahapatra
 - nnmin-aws
```

Comment:
can you remove Min's user name from the list here

Reply:
done